src/screens/ToDoList/styles.scss

```diff
@@ -0,0 +1,75 @@
+@import './../../shared/components/main-styles.scss';
+
+.container {
+  border: 3px solid $main-color;
+  border-radius: 18px;
+  width: 400px;
+  min-height: 400px;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 100px;
+  background-color:white;
+  display: flex;
+  flex-direction: column;
+  box-shadow: 5px 8px 5px 8px gray, 30px 10px 5px -90px black;
+}
+
+.text-decoration {
+  color:black;
+  display: flex;
+  letter-spacing: 2px;
+}
+
+.line {
+  border-bottom: 2px solid $main-color;
+  margin: 0 15px;
+  box-shadow: 1px 1px 1px 1px lightgray, 0px 0px 0px 0px lightgray;
+}
+
+.title {
+  align-items: center;
+  text-align: center;
+  color: $title-color;
+  font-stretch: expanded;
+  background-color: $main-color;
+  letter-spacing: 2px;
+}
+
+.clear-button {
+  width: 100px;
+  height: 30px;
+  margin: 13px 0px 10px 250px;
```

Comment:
You should use different approach to position button to the right. Setting margin to 250px is a bad idea.